ansible/playbooks/ansible.cfg

```diff
@@ -9,12 +9,17 @@ gathering = smart
 fact_caching = jsonfile
 fact_caching_connection = /tmp
 fact_caching_timeout = 86400
-stdout_callback = default
+# Use the YAML callback plugin because it's easier to read the errors
+stdout_callback = YAML
+# Use the stdout_callback when running ad-hoc commands.
+bin_ansible_callbacks = True
 display_skipped_hosts = no
 callbacks_enabled = profile_tasks
 deprecation_warnings = False
 inventory_ignore_extensions = ~, .orig, .bak, .ini, .cfg, .retry, .pyc, .pyo, .creds, .gpg, .example
 enable_plugins = host_list, script, auto, yaml, ini, toml
+# install and use local collections instead of global
+collections_paths = ./collections/ansible_collections
```

Comment:
I think this would put collections in a directory in playbooks, can you add that to our gitignore? 

RE: https://github.com/rackerlabs/genestack/pull/54/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R120

Reply:
You're already ignoring all directories called "ansible_collections" anywhere in your repo.

Reply:
https://github.com/rackerlabs/genestack/blob/218ec62056a8e9198dd202f1a577b83ba0bae5f8/.gitignore#L121

Reply:
You're correct, we're ignoring "ansible_collections;" however, this change is creating a new directory named "collections" which is where "ansible_collections" will reside, I'd like to add "collections" to our `.gitignore` so that nothing in that path is accidently committed.

Reply:
Pushed.